stories/content.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Content from './../src/web/components/Content';
+
+// TODO remove html entities when they are implemented
+const content = [
+  <h1>Title</h1>,
+  <p>Paragraph</p>,
+  <h2>Subtitle</h2>,
+  <ul>
+    <li>item list 1</li>
+    <li>item list 1</li>
+  </ul>,
+  <blockquote>blockquote</blockquote>,
+];
+
+storiesOf('Content', module)
+  .add('with small size', () => <Content size="small">{content}</Content>)
+  .add('with normal size', () => <Content>{content}</Content>)
```

Comment:
What is the difference between normal and medium ?

Reply:
the medium size is larger than the normal size. bulma gives us 4 sizes : Small > Normal (without css class) > medium > large